apps/donations/worker/wrangler.jsonc

```
@@ -112,6 +112,61 @@
         ],
       },
     },
+    "preview": {
+      "route": "https://donation-manager.preview.workers.alveussanctuary.org",
+
+      "vars": {
+        "SANDBOX": "true",
+        "SITE_URL": "https://alveusgg-private-git-private-preview-alveus.vercel.app",
```

```suggestion
        "SITE_URL": "https://preview.alveus.gg",
```
Can we point this at the main preview now?